<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewTodoDialog</class>
 <widget class="QDialog" name="AddNewTodoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>新建待办分组</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>分组名</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="NewLineEdit" name="groupNameLineEdit">
     <property name="inputMask">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>214</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="confirmBtn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="cannelBtn">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>240</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>154</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NewLineEdit</class>
   <extends>QLineEdit</extends>
   <header>newlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
